index.html

```diff
@@ -40,6 +40,19 @@ <h2 class="header-title">Welcome</h2>
             </span>
           </nav>
         </div>
+        <!-- Sidebar -->
+        <aside class="sidebar">
+          <div class="sidebar-header">
+            <span class="close-btn">
+              <i class="fas fa-times"></i>
+            </span>
+          </div>
+          <ul class="navbar">
+            <li><a href="#portfolio" class="portfolio">Portfolio</a></li>
+            <li><a href="#about" class="about">About</a></li>
+            <li><a href="#contact" class="contacts">Contact</a></li>
+          </ul>
```

Comment:
- [ ] Kindly align the nav link elements to the left side as it is in the provided Figma design.

**This is what your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/188504949-1d68f11f-84b9-4ec1-9966-93cdbdd7f903.png)

**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/188504990-5e186034-7cd4-435c-9979-85aa134e552d.png)